auth/authpb/auth.proto

```diff
@@ -15,3 +15,10 @@ message User {
   bytes password = 2;
   int64 tombstone = 3;
 }
+
+// Role is a single entry in the bucket authRoles
+message Role {
+  int64 ID = 1;
```

Comment:
We could, probably by adding an expensive version of role delete which also traverse users to remove attached roles. also we can build internal index to speed this up if we want.